src/main/plugins/org.dita.pdf2/xsl/fo/tables.xsl

```diff
@@ -399,7 +412,16 @@ See the accompanying LICENSE file for applicable license.
         </fo:table-row>
     </xsl:template>
 
+    <xsl:template match="*[contains(@class, ' topic/entry ')]" mode="validate-entry-position">
+        <xsl:if test="dita-ot:get-entry-end-position(.) &gt; number(ancestor::*[contains(@class,' topic/tgroup ')][1]/@cols)">
```

Comment:
Use `gt` instead of `>`

Reply:
Not sure what device you're viewing this on, but I wonder if it's converting -- I've currently got the `&gt;` entity rather than the greater-than character `>` ...

Reply:
Just `gt` not `&gt;`

Reply:
I mean use the `gt` operator from XPath 2.0 instead of `>` operator from XPath 1.0

Reply:
Ahh of course.